test/derivative_tests.jl

```diff
@@ -32,7 +32,20 @@ using ForwardDiff
 
                 # test jacobian to a Rotation matrix
                 R_jac = Rotations.jacobian(Quat, spq)
-                FD_jac = ForwardDiff.jacobian(x -> (q = Quat(SPQuat(x[1],x[2],x[3]));
+                FD_jac = ForwardDiff.jacobian(x -> (q = convert(Quat, SPQuat(x[1],x[2],x[3]));
```

Comment:
After #72, `convert(Quat, spquat)` and `Quat(spquat)` should do the same thing. No change needed here, just FYI.

Reply:
Thanks for the heads up. I'll work on switching over to that once I get used to the new (and improved) behavior.